src/components/Form.js

```diff
@@ -78,17 +82,12 @@ class Form extends Component {
   };
 
   handleSubmit = () => {
-    const { history, handleDrawerState, fromDrawer } = this.props;
-    let report = this.state;
+    let {thanksOpen, ...report} = this.state;
     delete report['media'];
-
     return axios.post(addReportUrl, report)
       .then(response => {
         if (response.status === 200) {
```

Comment:
Would we need to handle errors or the wait period ? For example, display circular progres until we receive the OK status or an error message on some network or other failures?

Reply:
Good point -- I've handled these now :)